doc/source/rllib/rllib-catalogs.rst

```diff
@@ -0,0 +1,151 @@
+.. include:: /_includes/rllib/rlmodules_rollout.rst
+
+.. _rllib-catalogs-user-guide:
+
+Catalogs
+========
+
+Catalogs are where RL Modules primarily get their models and action distributions from.
```

Comment:
The `Catalog` is a tool that helps create default models and action distributions for RLModules to work with environments with any action and observation spaces. It provides an interface for constructing pre-made components like an MLP for policies that have continuous action spaces, and an action distribution (like a multivariate Gaussian) that matches that action space.